packages/screens/Mini/components/MiniCommentInput/CommentTextInput.tsx

```diff
@@ -185,6 +185,7 @@ export const CommentTextInput = React.forwardRef<MiniCommentInputHandle, Props>(
                     );
                   }
                 }}
+                showSmallPreview
```

Comment:
In mini comment I used showSmallPreview flag to decrease height of selected image and video as using height as 400 with on screen keyboard would mess with UI